gis/tilemaker/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           boost 1.0
+
+github.setup        systemed tilemaker 2.2.0 v
+revision            0
+categories          gis
+license             FTWPL
+maintainers         {@sikmir disroot.org:sikmir} openmaintainer
+
+description         Make OpenStreetMap vector tiles without the stack
+long_description    {*}${description}
+
+checksums           rmd160  0843f7101651e3b5047ba539bdd4fdd7f9bde11e \
+                    sha256  74b0cc2f0cad5599ae30ceae5d4abbeefada30f0da5e0049efdbf6e7b0dd236a \
+                    size    42969929
+
+depends_lib-append  port:lua \
+                    port:protobuf3-cpp \
+                    port:rapidjson \
+                    port:shapelib \
+                    port:sqlite3 \
+                    port:zlib
+
+compiler.cxx_standard 2014
+
+post-destroot {
+    set mandir ${destroot}${prefix}/share/man
+    xinstall -m 755 -d ${mandir}/man1
+    xinstall -m 644 ${worksrcpath}/docs/man/tilemaker.1 ${mandir}/man1/
+}
```

Comment:
I *think* that MacPorts already creates the `man` directory and, if so, some of this is redundant. But I haven't had the time to double check yet....

Reply:
Yes indeed, fixed.